### DISCUSSION TITLE
linuxserver/netbox:latest does not show documentation

### DISCUSSION THREAD

**Discussion by jkoopmann** [netbox-community/netbox#12156] [Help Wanted!] [answered]
Hi,

just started with a fresh docker based netbox install. Netbox itself is running but when I try to access the "inline help" all I get is page not found.

e.g. 
http://mynetboxserver.local/static/docs/models/dcim/device/

I already ran mkdocs build in /app/netbox. Unfortunately this did not help. Any help would be greatly appreciated.

**Comment by candlerb** [comment, marked as answer]
The official community Netbox docker image is "netboxcommunity/netbox", and its build repo is [here](https://github.com/netbox-community/netbox-docker). If you are using a third-party image, then I suggest you raise issues with whoever builds and distributes that image.

> I already ran mkdocs build in /app/netbox

The netboxcommunity docker image doesn't use `/app`, it uses `/opt/netbox` as normal. Also, it *does* automatically run mkdocs as part of the [build process](https://github.com/netbox-community/netbox-docker/blob/release/Dockerfile#L93-L94).

**Reply by jkoopmann** [reply]
I was not aware that the other one was the "official" one. I switched to it (however was too stupid to find the documentation of the environment variables but after some analysis of the configuration.py transition was smooth). Thanks.
